api/service/syncing/syncing.go

```diff
@@ -32,7 +32,7 @@ const (
 	TimesToFail                     = 5 // downloadBlocks service retry limit
 	RegistrationNumber              = 3
 	SyncingPortDifference           = 3000
-	inSyncThreshold                 = 0    // when peerBlockHeight - myBlockHeight <= inSyncThreshold, it's ready to join consensus
+	inSyncThreshold                 = 1    // when peerBlockHeight - myBlockHeight <= inSyncThreshold, it's ready to join consensus
```

Comment:
why this change?

Reply:
see the todo added. because block sync will make the validator fail to accept consensus message. and in the rare case when the sync check happens in a bad timing, the validator could enter sync mode prematurely and miss consensus messages.